L18-jdbc/homework/src/main/java/ru/otus/jdbc/mapper/UniversalResultSetMapper.java

```diff
@@ -0,0 +1,59 @@
+package ru.otus.jdbc.mapper;
+
+import java.lang.reflect.InvocationTargetException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Arrays;
+import lombok.val;
+
+@SuppressWarnings("unchecked")
+public class UniversalResultSetMapper<T> {
+    private final EntityClassMetaData<T> entitySQLMetaData;
```

Comment:
`entityClassMetaData` ?
```suggestion
    private final EntityClassMetaData<T> entityClassMetaData;
```

Reply:
добавил геттер и парамертизацию да перемудрил